packages/webpack-config/src/helpers/paths.js

```diff
@@ -62,9 +58,29 @@ export const appTarget = resolveApp(config.destPath);
 
 const getAppName = () => {
   const { namespace } = getAppConfig();
-  if (namespace ==='') return 'app';
-  
+  if (namespace === "") return "app";
+
   return `${namespace}.app`;
-}
+};
+
+export const appName = getAppName();
+
+// check if a hbs partial dir is provided
+
+/******************************************************************************
+ ** CompileHTML helper
+ ******************************************************************************/
+export const hbsPartialDir = {
+  partialDirs: config.hbsPartialDir ? [resolveApp(config.hbsPartialDir)] : []
+};
+
+const getRequiredHtmlCompilerConfig = () => {
+  return Boolean(config.hbsEntry && config.hbsTarget);
+};
+export const useHtmlCompiler = getRequiredHtmlCompilerConfig();
+export const hbsEntry = useHtmlCompiler ? resolveApp(config.hbsEntry) : "/";
+export const hbsTarget = resolveApp(config.hbsTarget);
```

Comment:
I think this should be `const hbsTarget = useHtmlCompiler ? resolveApp(config.hbsTarget) : "/";` , (or a default value should be defined), otherwise the scripts won't run when no hbsTarget is provided in pv.config.js:

Reply:
added usecompilehtml check and fallback to "/"